_posts/en/newsletters/2020-04-29-newsletter.md

```diff
@@ -0,0 +1,265 @@
+---
+title: 'Bitcoin Optech Newsletter #95'
+permalink: /en/newsletters/2020/04/29/
+name: 2020-04-29-newsletter
+slug: 2020-04-29-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes the disclosure of an issue affecting
+the safety of routed LN payments, announces a new presigned vault
+proposal, and publishes a field report from Suredbits about building a
+high availability LN node.  Also included are our regular sections with
+popular questions and answers from the Bitcoin StackExchange,
+announcements of releases and release candidates, and descriptions of
+notable code changes in popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Review the disclosure of a potential LN issue:** as explained in the
+  *news* section below, a public disclosure this week describes a new
+  method for stealing money from LN routing nodes (unaffected are
+  endpoint nodes that are only the initial spenders or final receivers
+  of payments).  The issue partly overlaps with a previous well-known
+  issue that has not been exploited (to our knowledge) because almost
+  all onchain transactions relayed in the past two years confirmed
+  relatively quickly even if they only paid the default minimum relay
+  feerate.  If feerates increase significantly for an extended period of
+  time, these issues will become more critical.  See our explanation in
+  the *news* section for details and contact your LN software vendor if
+  you have concerns about how the this issue might affect your channels.
+
+## News
+
+<!-- break atomicity == fission attack? -->
```

Comment:
That's a nice finding, even if consequence is effectively to break settlement atomicity, attack relies more on blinding victim node information availability. 

In LN, events triggering state transition are assumed to be announced privately between parties or published globally on the blockchain. Pinning relies on depriving a honest party to commit an onchain event while blinding it to observe the reason of failure, adding a mempool-dimension between offchain and onchain. 

Mempools aren't consistent, it's like in quantum mechanics, you don't have a neutral observer, locality of observation changes the phenomenon observed. But don't have a name to catch the idea :)